test/src/cookies.spec.ts

```diff
@@ -379,6 +379,34 @@ describe('Cookie specs', () => {
       await page.goto(server.PREFIX + '/grid.html');
       expect(await page.evaluate('document.cookie')).toBe('gridcookie=GRID');
     });
+    it('should set a cookie with a partitionKey', async () => {
```

Comment:
so this test has to be something like this:

```
    it('should set a cookie with a partitionKey', async () => {
      const {page, server, isChrome} = await getTestState();

      await page.goto(server.EMPTY_PAGE);
      const url = new URL(page.url());
      await page.setCookie({
        url: url.toString(),
        name: 'partitionCookie',
        value: 'partition',
        secure: true,
        partitionKey: url.origin,
      });
      await expectCookieEquals(await page.cookies(), [
        {
          name: 'partitionCookie',
          value: 'partition',
          domain: url.hostname,
          path: '/',
          expires: -1,
          size: 24,
          httpOnly: false,
          secure: true,
          session: true,
          sameParty: false,
          sourceScheme: 'Secure',
          partitionKey: isChrome ? url.origin.replace(`:${url.port}`, '') : url.origin,
        },
      ]);
    });
```

unfortunately, there are some differences in partitioning across browsers.

Reply:
done